spago.yaml

```
@@ -62,3 +63,106 @@ workspace:
       git: https://github.com/purescript/registry-dev.git
       ref: 6a803c37577af368caa221a2a06d6be2079d32da
       subdir: foreign
+    html-parser-halogen:
+      dependencies:
+        - halogen
+        - string-parsers
+      git: https://github.com/rnons/purescript-html-parser-halogen.git
+      ref: 035a51d02ba9f8b70c3ffd9fe31a3f5bed19941c
+    jest:
+      dependencies:
+        - aff
+        - aff-promise
+        - effect
+        - foldable-traversable
+        - prelude
+        - psci-support
+      git: https://github.com/klntsky/purescript-jest.git
+      ref: 7feaa5a880fc75002c4eca312993174e7220252b
+    literals:
+      dependencies:
+        - assert
+        - console
+        - effect
+        - integers
+        - numbers
+        - partial
+        - psci-support
+        - typelevel-prelude
+        - unsafe-coerce
+      git: https://github.com/jvliwanag/purescript-literals.git
+      ref: ae3ef4e9c1ae7c57ec77bd13906fa60ae8abba4a
+    markdown-it:
+      dependencies:
+        - effect
+        - options
+        - prelude
+      git: https://github.com/klntsky/purescript-markdown-it.git
+      ref: f3b7654783a83a80d7c09b6caaa7cd40b93ddce1
```

We can move back to the upstream here:
```suggestion
      git: https://github.com/nonbili/purescript-markdown-it.git
      ref: f6e8ee91298f2fc13c4277e75a19e0538de5f7a2
```